src/main/java/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction.java

```diff
@@ -129,6 +156,111 @@ public BuildReference(final String projectName) {
       this.buildNumber = 0;
       this.buildResult = Result.NOT_BUILT;
     }
+
+    public String getProjectName() {
+      return projectName;
+    }
+
+    public int getBuildNumber() {
+      return buildNumber;
+    }
+
+    public Result getBuildResult() {
+      return buildResult;
+    }
+
+    @Override
+    public void update() {
+
+    }
+  }
+
+  public static class StaticBuildReference extends AbstractBuildReference {
```

Comment:
This looks identical to the BuildReference.  Is there a reason to use this over that?